aidants_connect_web/models.py

```diff
@@ -746,7 +751,7 @@ def revocation_date(self) -> Optional[datetime]:
         """
         return (
             self.autorisations.order_by("-revocation_date").first().revocation_date
-            if self.was_explicitly_revoked
+            if self.was_explicitly_revoked and self.autorisations.all().exists()
```

Comment:
Est-ce qu'on a besoin du `.all()` ? J'ai un doute.

```suggestion
            if self.was_explicitly_revoked and self.autorisations.exists()
```